neuralprophet/utils.py

```diff
@@ -806,35 +806,71 @@ def configure_trainer(
 
     # Configure callbacks
     callbacks = []
+    has_custom_callbacks = True if "callbacks" in config else False
 
     # Configure checkpointing
-    if checkpointing_enabled:
-        # Callback to access both the last and best model
-        checkpoint_callback = pl.callbacks.ModelCheckpoint(
-            monitor=early_stopping_target, mode="min", save_top_k=1, save_last=True
+    has_modelcheckpoint_callback = (
+        True
+        if has_custom_callbacks and any(isinstance(x, pl.callbacks.ModelCheckpoint) for x in config["callbacks"])
```

Comment:
Use self-explanatory variable name
```suggestion
        if has_custom_callbacks and any(isinstance(callback, pl.callbacks.ModelCheckpoint) for callback in config["callbacks"])
```